### PR TITLE
Add payout request API

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

File: openapi/paths/payout-requests@{id}@payment-instruments.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+get:
+  x-products:
+    - Core
+  tags:
+    - Transactions
+  summary: Retrieve payout request's payment instruments
+  operationId: GetPayoutRequestPaymentInstruments
+  x-sdk-operation-name: getPaymentInstruments
+  description: |-
+    Retrieves a list of payment instruments with associated gateway of a payout request with specified ID.
```
Comment (noon-dawg) [inline openapi/paths/payout-requests@{id}@payment-instruments.yaml:12]: nitpick: avoid using block scalars for single line descriptions
```suggestion
  description: Retrieves the payment gateway and a list of payment instruments that are associated with a payout request with a specified ID.
```

File: openapi/paths/payout-requests@{id}@payment-instruments.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            items:
+              type: object
+              properties:
+                paymentInstrumentId:
+                  description: ID of the payment instrument to offer.
+                  type: string
+                gatewayName:
+                  description: |-
+                    Name of the payment gateway to use for the payout along the payment instrument.
+                    Empty for the requested payment instrument if it hasn't been used yet.
```
Comment (noon-dawg) [inline openapi/paths/payout-requests@{id}@payment-instruments.yaml:31]: nitpick: suggestion for readability 
```suggestion
                    Name of the payment gateway and payment instrument to use for the payout.
                    This field is empty is the requested payment instrument has not been used yet.
```

File: openapi/components/schemas/PayoutRequest.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            - $ref: ./ServerTimestamp.yaml
+        updatedTime:
+          $ref: ./UpdatedTime.yaml
+  reason:
+    description: |-
+      Rejection or cancellation reason of the payout request for "rejected" or "canceled" status appropriately.
+    type: string
+    readOnly: true
+  reviewerId:
+    description: ID of the reviewer user.
```
Comment (noon-dawg) [inline openapi/components/schemas/PayoutRequest.yaml:100]: nitpick: I don't think we need 'user' here
```suggestion
    description: ID of the reviewer.
```

**Review by noon-dawg — COMMENTED** [review]
Excellent docs!

All my feedback is minor

File: openapi/components/schemas/PayoutRequest.yaml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    type: string
+    nullable: true
+    readOnly: true
+    allOf:
+      - $ref: ./ResourceId.yaml
+  reviewTime:
+    description: Review time of the payout request.
```
[the remark this answers is not in the document]
Comment (adamaltman) [reply openapi/components/schemas/PayoutRequest.yaml:107]: "was" will violate a rule.

[the remark this answers is not in the document]
Comment (adamaltman) [reply openapi/components/schemas/PayoutRequest.yaml:107]:
```suggestion
    description: Date and time when the payout request is reviewed.
```

[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/PayoutRequest.yaml:107]: Good point. We should use `is` for all of these. I'll update the other comments

File: openapi/paths/payout-requests@{id}@payment-instruments.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            items:
+              type: object
+              properties:
+                paymentInstrumentId:
+                  description: ID of the payment instrument to offer for the payout.
+                  type: string
+                gatewayName:
+                  description: |-
+                    Name of the payment gateway and payment instrument to use for the payout.
+                    This field is empty is the requested payment instrument has not been used yet.
```
Comment (noon-dawg) [inline openapi/paths/payout-requests@{id}@payment-instruments.yaml:30]: typo:
```suggestion
                    This field is empty if the requested payment instrument has not been used yet.
```

**Review by noon-dawg — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonTransactionRequest.yaml [hunk 6]
```diff
@@ -96,3 +96,7 @@ properties:
       Time the transaction is processed.
       Can be specified only if the transaction is processed outside Rebilly.
     format: date-time
+  payoutRequestId:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonTransactionRequest.yaml:99]: Shouldn't it be added to https://github.com/Rebilly/api-definitions/blob/26dd41e73c5010ecc8e509d107202c8491dd8654/openapi/components/schemas/Transactions/PayoutRequest.yaml#L1?

**Comment by 2e3s** [comment]
devs please review

**Comment by ashkarpetin** [comment]
lgtm

**Review by 2e3s — DISMISSED** [review]

File: openapi/paths/payout-requests@{id}@payment-instruments.yaml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  description: Retrieves the payment gateway and a list of payment instruments that are associated with a payout request with a specified ID.
+  responses:
+    200:
+      description: Payout request associated payment instruments retrieved.
+      content:
+        application/json:
+          schema:
+            type: array
+            description: |-
+              List of available payment instruments per an associated gateway to offer for the payout.
```
Comment (noon-dawg) [inline openapi/paths/payout-requests@{id}@payment-instruments.yaml:20]: nitpick: we should avoid adding block scalars for one liners
```suggestion
            description: List of available payment instruments per an associated gateway to offer for the payout.
```

**Review by 2e3s — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Pull request merged**